CesiumJsonReader/include/CesiumJsonReader/ArrayJsonHandler.h

```diff
@@ -453,4 +453,112 @@ class CESIUMJSONREADER_API ArrayJsonHandler<std::string, StringJsonHandler>
   std::vector<std::string>* _pArray = nullptr;
   bool _arrayIsOpen = false;
 };
+
+template <typename T, typename THandler>
+class CESIUMJSONREADER_API
+    ArrayJsonHandler<std::vector<T>, ArrayJsonHandler<T, THandler>>
+    : public JsonHandler {
+public:
+  using ValueType = std::vector<T>;
+
+  template <typename... Ts>
+  ArrayJsonHandler(Ts&&... args) noexcept
+      : JsonHandler(),
+        _handlerFactory(
+            std::bind(handlerFactory<Ts...>, std::forward<Ts>(args)...)),
+        _elementHandler() {}
+
+  void reset(IJsonHandler* pParent, std::vector<std::vector<T>>* pArray) {
+    JsonHandler::reset(pParent);
+    this->_pArray = pArray;
+    this->_arrayIsOpen = false;
+    this->_elementHandler.reset(this->_handlerFactory());
+  }
+
+  virtual IJsonHandler* readNull() override {
+    return this->invalid("A null")->readNull();
+  }
+
+  virtual IJsonHandler* readBool(bool b) override {
+    return this->invalid("A bool")->readBool(b);
```

Comment:
This was originally "A null", which I assumed was a typo. I've changed it, and another occurrence in this file.